### ISSUE TITLE
Problem in reproduce 

### ISSUE THREAD

**Issue by HITKJ** [wayveai/mile#29]
Thans for your excellent work. I have attempted to reproduce your results, and below are the comparison results between the reproduced model and the model you provided.There is a big gap between my results and yours. Hope you can provide some guidance.
<img width="272" alt="4731c200d3db5b713fd322936c861692" src="https://github.com/wayveai/mile/assets/50793811/ce86e837-a242-4c2f-845e-7abd011887eb">

 I collected data on four different maps (town01, town03, town04, town06) at 10 FPS for 25 episodes on each map. In each episode, I collected 3000 images, resulting in nearly 300,000 images for training data. During training,  I used a batch size of 64 for a total of 50,000 iterations at 10 FPS and also tested at 10 FPS. Additionally, I used 1000 batches validation set  at every 3000 iteration. The validation data setup was consistent with the simulation test, and the validation metrics showed a decreasing trend during the iterations.
<img width="424" alt="image" src="https://github.com/wayveai/mile/assets/50793811/2ef895b6-df5f-4f46-81c9-322f23e0ab3f">
<img width="406" alt="image" src="https://github.com/wayveai/mile/assets/50793811/e59a68b6-d117-4830-bff7-5b2bc1d0442f">

Another question, the test time is too long, can we use multiple gpus for acceleration.
